src/bpftrace.h

```diff
@@ -51,10 +52,12 @@ inline DebugLevel operator++(DebugLevel& level, int)
 class BPFtrace
 {
 public:
-  BPFtrace() : ncpus_(get_possible_cpus().size()) { }
+  BPFtrace() : out_(std::cout),ncpus_(get_possible_cpus().size()) { }
+  BPFtrace(std::ostream& o) : out_(o),ncpus_(get_possible_cpus().size()) { }
   virtual ~BPFtrace();
   virtual int add_probe(ast::Probe &p);
   int num_probes() const;
+  std::ostream& outputstream() { return out_; }
```

Comment:
I think this function can be made `const`

Reply:
`const ostream` isn't possible, or did you mean something else?

Reply:
Sorry, I meant `std::ostream& outputstream() const {...}`, not 100% sure about this though

Reply:
This seems to work, the runtime tests pass.